cross/ffmpeg/patches/034-avcodec-arm-sbcenc-avoid-callee-preserved-vfp-regist.patch

```diff
@@ -0,0 +1,287 @@
+From: James Cowgill <jcowgill@debian.org>
+Date: Sun, 11 Aug 2019 16:50:56 +0100
+Subject: avcodec/arm/sbcenc: avoid callee preserved vfp registers
+
+When compiling FFmpeg with GCC-9, some very random segfaults were
+observed in code which had previously called down into the SBC encoder
+NEON assembly routines. This was caused by these functions clobbering
```

Comment:
Not sure of the pertinence of this patch:
1. at the very least, this only affects arm as such it should be under `patches/arm` directory.  As NEON is not for all arm arch it may actually require to be under `patches/armv8` (and perhaps also copied over under `patches/armv7` and `patches/armv7l`, unsure)
2. Secondly, it associate the random segfaults to GCC9 but DSM6 uses gcc4 and DSM7 uses gcc5 so in theory this doesn't apply

Reply:
2) Well applying this at least didnt segfault the build ;)

Reply:
At the very very least, please move that under a `patches/arm` folder to not be affecting non-arm builds.

Reply:
Done.